packages/frontend/src/components/common/PrimaryLinkButton.tsx

```diff
@@ -0,0 +1,50 @@
+import { Anchor, Button, ButtonProps } from '@mantine/core';
+import React, { FC } from 'react';
+import { useHistory } from 'react-router-dom';
+import { EventData, useTracking } from '../../providers/TrackingProvider';
+
+export interface PrimaryLinkButtonProps extends ButtonProps {
+    href: string;
+    trackingEvent?: EventData;
+    target?: React.HTMLAttributeAnchorTarget;
+    forceRefresh?: boolean;
+    onClick?: (e: React.MouseEvent<HTMLButtonElement>) => void;
+}
+
+const PrimaryLinkButton: FC<PrimaryLinkButtonProps> = ({
```

Comment:
I think `Primary` is misleading. you can name it MantineLinkButton or MantineAnchorButton before we get rid of Blueprint